Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/ExchangeManagerServiceImpl.java

```diff
@@ -59,22 +75,37 @@
     private final ExchangeManager exchangeManager = ExchangeManager.getInstance();
     private final PingService pingService = new PingServiceImpl();
     private static final String DATE_FORMAT = "MM-dd-yy hh:mm:ss";
+    private static final WebServiceProxyHelper oProxyHelper = new WebServiceProxyHelper();
 
     @Override
     public boolean saveExchange(ExchangeType exchange) {
+        SaveExchangeRequestMessageType request = new SaveExchangeRequestMessageType();
+        request.setConfigAssertion(buildConfigAssertion());
+        request.setExchange(exchange);
+
         try {
-            return exchangeManager.saveExchange(exchange);
-        } catch (ExchangeManagerException e) {
+            SimpleExchangeManagementResponseMessageType response = (SimpleExchangeManagementResponseMessageType) clientInvokePort(
+                ADMIN_EXCHANGE_SAVE_EXCHANGE, request);
+            LOG.debug("{}: {}, {}", ADMIN_EXCHANGE_SAVE_EXCHANGE, response.isStatus(), response.getMessage());
+            return response.isStatus();
+        } catch (Exception e) {
```

Comment:
Can we be more specific? Why do we need a Try/Catch here?

Reply:
exception is being throw by invokePort